core/src/main/resources/hudson/model/AbstractProject/main.jelly

```diff
@@ -34,7 +34,7 @@ THE SOFTWARE.
 
     <j:forEach var="act" items="${it.prominentActions}">
       <j:set var="icon" value="${act.iconClassName != null ? act.iconClassName : act.iconFileName}"/>
-      <t:summary icon="${icon}" href="${act.urlName}">
+      <t:summary icon="${attrs.icon}" href="${act.urlName}">
```

Comment:
This change to the display of prominent actions on the main project page (not the **Configure** project page) seems unrelated to the primary goal of this PR. In addition, it renders the line above it (which defines a local variable by the name of `icon`) dead code by virtue of relying on the `attrs` map rather than a local variable. Unused variables make a program more difficult to read and maintain and are generally discouraged by most programming style guides. If this change is unrelated to the primary goal of this PR, then I suggest reverting it and proposing it in a separate PR.

Reply:
Great spot, sorry that wasn't intentional. Had a bug with the icons during development and this must have been included in the commit by mistake.